1-3.md

```diff
@@ -81,13 +85,14 @@ Approximately 30 to 45 minutes.
 
 #### Teaching Ideas
 
-* Discuss how accessibility solutions are also enjoyed by many people regardless of their disability. Emphasize how many solutions originally thought for accessibility have become mainstream, such as the type-writer, telephone, punch cards, text to speech, email, and voice controls.
-* Promote discussion among students about which aspects of accessibility benefit them as well as people with disabilities. For instance, ask them which products they can use without looking at them or without pressing any button.
+* Discuss how accessibility features benefit many people in different situations, such as mobile phone users, older people, and people using different hardware. Explain that many features originally thought for accessibility have become mainstream, such as the type-writer, telephone, punch cards, text to speech, email, and voice controls.
+* Promote discussion among students about which accessibility features they may already be using and which they may want to use. For example, ask them if they have used voice to dictate or to listen to messages.
+* Provide analogies between digital accessibility, and social and architectural accessibility. For example, relate digital accessibility to the curb cut on sidewalk, which benefits people using strollers, bicycles, pushing carts, carrying luggage, etc. This concept is sometimes referred to as the electronic curb cut.
 
 #### Homework Ideas
 
-* Ask students to think about how many people with disabilities there might be in their organization. Emphasize that statistics on this matter are difficult to find and interpret. The concept of disability, the way it is measured in different cultures, and the right for people not to disclose their disabilities may influence the outcomes of this exercise.
```

Comment:
I'm very hesitant about such an exercise. This has severe privacy implication and likely to be very superficial (ie. only visible and diagnosed disabilities), which would skew any statistics anyway. How does that relate to innovation?